src/convertToObject.js

```diff
@@ -6,7 +6,22 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const formatedCSSData = {};
+  const splitedSource = sourceString.split(';');
+
+  splitedSource
+    .map((item) => {
+      const formatedPropAndValueObject = item
+        .split(':')
+        .map((elem) => elem.trim());
+
+      return formatedPropAndValueObject;
+    })
+    .forEach(([property, value]) => {
+      Object.assign(formatedCSSData, { [property]: value });
+    });
```

Comment:
The use of map followed by forEach is unnecessary and can be simplified. You can directly use forEach to iterate over 'splitedSource' and build the 'formatedCSSData' object without creating an intermediate array.